src/mocks/handlers.ts

```diff
@@ -0,0 +1,17 @@
+import { http, HttpResponse } from "msw";
+import { facilityData, mockAssetData } from "./assetCreateMockData";
+
+export const assetHandlers = [
+  // Handler for listing facility asset locations
+  http.get(
+    "https://careapi.ohc.network/api/v1/facility/:facility_external_id/asset_location/",
```

Comment:
I'm planning to create a helper function, so that we can directly use the paths from `api.ts`